src/Runners/PHPUnit/Options.php

```diff
@@ -58,6 +58,47 @@ class Options
     protected $filtered;
 
     /**
+     * @var string
+     */
+    protected $runner;
+
+    /**
+     * @var bool
+     */
+    protected $noTestTokens;
+
+    /**
+     * @var bool
+     */
+    protected $colors;
+
+    /**
+     * Filters which tests to run.
+     * @var ?string
```

Comment:
There are more of these nullable annotations. As you commented above, it's already on the phpdoc roadmap but I would suggest sticking with `string|null` until it's in. Probably this is only important for IDE based type detection as we don't have phpdoc in the ci IIRC

applies for every `\?[\w]+` in this diff ;)